caveclient/tools/table_manager.py

```diff
@@ -856,6 +861,7 @@ def query(
                     offset=offset,
                     select_columns=select_columns,
                     get_counts=get_counts,
+                    log_warning=log_warning,
```

Comment:
I might be blind, but I don't see `log_warning` a parameter in the `materialize.query_view` function right now: https://github.com/CAVEconnectome/CAVEclient/blob/884309e434db06a3908fb69c4ab433fbeba613d3/caveclient/materializationengine.py#L2356